ouroboros-network-api/src/Ouroboros/Network/NodeToClient/Version.hs

```diff
@@ -69,14 +72,15 @@ nodeToClientVersionCodec = CodecCBORTerm { encodeTerm, decodeTerm }
         (13, True) -> Right NodeToClientV_13
         (14, True) -> Right NodeToClientV_14
         (15, True) -> Right NodeToClientV_15
+        (16, True) -> Right NodeToClientV_16
         (n, _)     -> Left ( T.pack "decode NodeToClientVersion: unknown tag: " <> T.pack (show tag)
                             , Just n)
       decodeTerm _  = Left ( T.pack "decode NodeToClientVersion: unexpected term"
                            , Nothing)
 
 
 nodeToClientVersionBit :: Int
-nodeToClientVersionBit = 15
+nodeToClientVersionBit = 16
```

Comment:
Thank you for pointing this out. I've added a comment about this above the binding itself, to avoid future confusion. I also moved it inside the `were` clause from the top level to indicate that `nodeToClientVersionCodec` is the only function that uses it.